src/schema/schema.graphql

```diff
@@ -3169,6 +3169,39 @@ input SaveDocumentInput {
 	document: SaveDocumentInput
 }
 
+input DLActionAttrInput {
+	n: String
```

Comment:
we usually try to use more readable keys in schema and normalize/denormalize it as part of data layer

Reply:
Done.